Packs/AzureLogAnalytics/Integrations/AzureLogAnalytics/AzureLogAnalytics_description.md

```diff
@@ -2,18 +2,19 @@ Log Analytics is a service that helps you collect and analyze data generated by
 
 Full documentation for this integration is available in the [reference docs](https://xsoar.pan.dev/docs/reference/integrations/azure-log-analytics).
 
+There are two authentication methods available:
 
-## Authorize Cortex XSOAR for Azure Log Analytics
+ * [Cortex XSOAR Application](https://xsoar.pan.dev/docs/reference/articles/microsoft-integrations---authentication#cortex-xsoar-application)
+ * [Self-Deployed Application](https://xsoar.pan.dev/docs/reference/articles/microsoft-integrations---authentication#self-deployed-application)
 
-You need to grant Cortex XSOAR authorization to access Azure Log Analytics.
+Depending on the authentication method that you use, the integration parameters might change.
 
-1. Access the [authorization flow](https://oproxy.demisto.ninja/ms-azure-log-analytics). 
-2. Click the **Start Authorization Process** button. 
-   You will be prompted to grant Cortex XSOAR permissions for your Azure Service Management. 
-3. Click the **Accept** button. 
-   You will receive your ID, token, and key. You need to enter this information, when you configure the Azure Log Analytics integration instance in Cortex XSOAR.
+#### Cortex XSOAR Azure App
 
-## Authorize Cortex XSOAR for Azure Log Analytics - Self-Deployed Configuration
+To use the **Cortex XSOAR application** and allow Cortex XSOAR access to Azure Log Analytics, an administrator has to approve our app using an admin consent flow by clicking this **[link](https://oproxy.demisto.ninja/ms-azure-log-analytics)**.
```

Comment:
Not sure what you mean by "our app".  Please change the wording so there is no doubt what app you are referring to.

Reply:
like here: https://github.com/demisto/content/pull/31866#discussion_r1440524833